### ISSUE TITLE
Feature Request: Release notes / arbitrary "homepage" link

### ISSUE THREAD

**Issue by jim80net** [keel-hq/keel#227]
It'd be great to be able to supply a link to the maintainer's page so that I could review release notes, or sate my curiosity that yes, keel did get the latest version. 

I think it could be passed in when the image is configured as a string and then possibly parameterized with the version value, particularly for jumping to a particular release. 

```
keel:
  policy: all
  trigger: poll
  pollSchedule: "@every 3m"
  images:
    - repository: image.repository 
      tag: image.tag 
      url : https://github.com/keel-hq/keel/releases/tag/< SOME_TEMPLATE_PATTERN > 
```

Then, when I get a notification, the URL could be passed as well, providing for some awesome just in time context. =)

**Comment by rusenask** [comment]
very good idea! I will have a look for the easiest way to add this. I think we could extend TrackedImage struct to include this URL and then add it to the notification if it's set

**Comment by rusenask** [comment]
available in 0.9.5
